src/State/FluentState.php

```diff
@@ -75,11 +79,14 @@ public function getDomain()
     /**
      * Set the current domain code
      *
-     * @param string $domain
+     * @param string|null $domain
      * @return $this
```

Comment:
Can we add some PHPDocs for the exceptions this throws?

Reply:
We don't really do that in core; It could be a standard, but I'd probably wait until we did it consistently before adding it to other modules.

And then, I'd do it everywhere of course. :)

Reply:
Oh? Ok 😆